packages/react-components/src/components/SendBox.tsx

```diff
@@ -353,7 +353,12 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   }, [activeFileUploads, props, localeStrings]);
 
   return (
-    <Stack className={mergeStyles(sendBoxWrapperStyles)}>
+    <Stack
+      className={mergeStyles(
+        sendBoxWrapperStyles,
+        { overflow: 'visible' } // This is needed for the mention popup to be visible
```

Comment:
What would be happening if the content of senboxWrapper has real content overflow? Like too many line of text in the box?